linera-storage/src/db_storage.rs

```diff
@@ -813,3 +814,44 @@ where
         Ok(Self::create(storage, WallClock))
     }
 }
+
+#[cfg(with_testing)]
+impl<Client> DbStorage<Client, TestClock>
+where
+    Client: KeyValueStore + Clone + Send + Sync + 'static,
+    ViewError: From<Client::Error>,
+    Client::Error:
+        From<bcs::Error> + From<DatabaseConsistencyError> + Send + Sync + serde::ser::StdError,
+{
+    pub async fn make_test_storage(wasm_runtime: Option<WasmRuntime>) -> Self {
```

Comment:
This seems like a quick&dirty shortcut. Can we move this to `linera-core/src/unit_tests/wasm_worker_tests.rs` where it seems to be useful?

Reply:
That seems difficult as rust forces the implementation in the same file.

Reply:
For a change I meant as a helper function

Reply:
What would this achieve? We have the `make_test_storage` next to the `new_for_testing`. This would disperse the code for no visible gain.

Reply:
Do we need two APIs with inconsistent names?